freeglut/freeglut/src/fg_internal.h

```diff
@@ -572,9 +670,10 @@ do                                            \
 {                                             \
```

Comment:
Ok, VS2015 compiles it fine. I don't know about older compilers. I'm writing an email for this right now.